.drone.yml

```diff
@@ -19,13 +19,6 @@ steps:
     commands:
       - hadolint Dockerfile
 
-  - name: lint_helm_files
-    image: pelotech/drone-helm3
-    settings:
-      mode: lint
-      chart: ./helm
-      values_file: ./helm/values.yaml
```

Comment:
Why are we not linting `./helm/xxx-values.yaml` files

Reply:
Resolving as it isn't possible to lint files with the pelotech/drone-helm3 plugin without a local path to Chart.yaml ([link to docs](https://github.com/pelotech/drone-helm3/blob/master/docs/parameter_reference.md#linting))